src/components/CreateApplication/type.ts

```diff
@@ -0,0 +1,4 @@
+export type CreateApplicationValues = {
```

Comment:
This could be part of the form component itself if its not being reused by any other component.

Reply:
fixed this